pkg/resources/flow.go

```diff
@@ -135,17 +133,38 @@ func (f *Flow) Create() error {
 // Delete is called during dlow destruction which can happen only once while Create ensures that at least one flow
 // replica exists, and as such can be called any number of times
 func (f Flow) Delete() error {
-	graph, err := f.buildDependencyGraph(true)
+	graph, err := f.buildDependencyGraph(-1, 0, false)
 	if err != nil {
 		return err
 	}
 	stopChan := make(chan struct{})
 	graph.Deploy(stopChan)
-	f.status = interfaces.ResourceReady
 	return nil
 }
 
 // Current status of the flow deployment
```

Comment:
Can you fix docstring in the same PR? If there is a change in content of func - we can also fix docstring, as it's a good occasion ;)
Btw. same for `Delete` above.